### ISSUE TITLE
WslRegisterDistribution failed

### ISSUE THREAD

**Issue by maoxuner** [VSWSL/Arch-WSL#24]
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x8007019e
Error: 0x8007019e ??????? Linux ? Windows ????

Press any key to continue...

**Comment by kepet19** [comment]
Do you have this enabled?
![image](https://github.com/VSWSL/Arch-WSL/assets/55736468/47ec2949-4415-421a-a0f3-4e2952a6c7f8)

To open the window
`Win + R`
`OptionalFeatures.exe`


After that is installed and rebooted. 
I would recommend to run this command once `wsl --set-default-version 2`
To set the WSL environment to WSL2.

Then go to the Microsoft store, search for the package and press open.

**Comment by maoxuner** [comment]
sorry, i didn't enable subsystem feature.

**Comment by kepet19** [comment]
> sorry, i didn't enable subsystem feature.

No problem.
I had the same issue yesterday. 
I then ran into another problem when using git to clone a repo. 
Then I came to the issues looking for the problem/solution. 
I saw your issue and would like to help. 


The other issue was me trying to clone a git repo using ssh, without installing ssh. 
It then used windows ssh.exe and gave a weird error that said CMD.exe could not do something about UNC paths. 
I did not understand the issue at first. 
I kept running `which` `whereis` command for `git`, and could not understand it. It is all on the Linux system, why is it using CMD.exe when I am using git? 
Then I released I maybe did not have ssh installed on WSL.
